website/src/components/Sidebar.js

```diff
@@ -1,10 +1,23 @@
-import React from 'react'
+import React, { useState } from 'react'
+import SearchBar from './SearchBar';
 
 function Sidebar ({children = []}) {
+    const [searchTerm,setSearchTerm] = useState(''); 
+    // console.log(children)
+    const filtered = 
+    children.filter(val => {
+        if (searchTerm === '') {
```

Comment:
I think if search term is empty then all values should be shown. So, you can return true right away without testing `val.props.text || val.props.children[0].props.text`. Unless there's a reason I'm not aware that it's necessary?